docs/source/test_network.md

```diff
@@ -376,7 +376,7 @@ local machine, you can use the tutorials to start developing your own solution:
 
```

Comment:
The top of the Test Network tutorial under Before you begin still refers to to a topic titled  "`Installed the Samples, Binaries and Docker Images.` which I believe is now renamed. 


Before you can run the test network, you need to clone the fabric-samples repository and download the Fabric images. Make sure that you have installed the Prerequisites and Installed the Samples, Binaries and Docker Images.

Reply:
updated.